user-programs/folk-baltimore/sounds/README.md

```diff
@@ -0,0 +1 @@
+- [folk_boot_guitar.wav](https://freesound.org/people/SergeQuadrado/sounds/817421/) by SergeQuadrado is licensed under a Creative Commons Attribution NonCommercial 4.0 License
```

Comment:
I can remove this file and the WAV file from the final commit - mostly I wanted to make it easier to test